azure/packages/test/end-to-end-tests/src/test/audience.spec.ts

```diff
@@ -111,9 +113,12 @@ describe("Fluid audience", () => {
             errorMsg: "container connect() timeout",
         });
 
-        const client2 = createAzureClient("test-id-2", "test-user-name-2");
+        const client2 = createAzureClient("test-user-id-2", "test-user-name-2");
         const { services: servicesGet } = await client2.getContainer(containerId, schema);
 
+        const partner = await waitForMyself(servicesGet.audience, "test-user-id-2");
```

Comment:
I think this await is working around the issue that getContainer returns before myself is in the audience?  If so, maybe add a comment here explaining that (otherwise I would expect I could immediately getMyself)

Reply:
Good point. We had it called out at few other places in this file, but it does not hurt to repeat it across the board.